pynpoint/readwrite/nearreading.py

```diff
@@ -365,6 +392,20 @@ def run(self) -> None:
             # get the images of chop A and chop B
             chopa, chopb = self.read_images(filename, im_shape)
 
+            if self.m_do_chop_sub:
+                chopa = chopa - chopb
```

Comment:
In the `SubtractImageModule` there is a scaling factor applied. Is that not necessary here?

Reply:
That's correct, the scaling factor isn't necessary for the chop subtraction, so I think fixing that parameter to 1 in the `NEARReadingModule` is ok.